graphql/database/users/createUser.js

```diff
@@ -72,9 +68,10 @@ const createUser = async (
 
   // Set default background image to a cat image if user is enabled for v4 beta
   if (v4BetaEnabled) {
+    const { backgroundImageCategory } = await getCause(causeId)
```

Comment:
merged in Homepage code that sends up the causeId.  I'm not fully sure about why this is idempotent but I think that createUser is always called from the auth front end and so that value should always be set in local storage.  Is this right?

Reply:
No, it won't necessarily exist. `createUser` is called every time a user logs in, I believe, and their local storage could have cleared since they joined.

I'm wondering, can we just get rid of this logic here altogether and rely on the `getBackgroundImage` resolver to set an image (by category) if one is not already set? We already do a lot of this work elsewhere—no real reason for the data to be set on user creation. Thoughts? I might be missing something, so push back if you feel otherwise.

Reply:
I liked this idea.  I pushed that move, I think it's pretty comprehensive.  Lmk what you think